Doc/whatsnew/3.8.rst

```diff
@@ -1575,7 +1633,7 @@ Deprecated
 
 * Passing an object that is not an instance of
   :class:`concurrent.futures.ThreadPoolExecutor` to
-  :meth:`asyncio.loop.set_default_executor()` is
+  :meth:`asyncio.loop.set_default_executor` is
```

Comment:
Minor sphinx role fix.

Reply:
If you want to aim for perfection you can do `:meth:`loop.set_default_executor() <asyncio.loop.set_default_executor>`  (that way it reads better)

Reply:
@1st1 Even without the parens `()` it includes them after the role has been rendered. See the current version:

![image](https://user-images.githubusercontent.com/44193521/67451297-023bb600-f5ee-11e9-8daa-86707ca05162.png)

So, 
```
:meth:`loop.set_default_executor() <asyncio.loop.set_default_executor>`
```
and
```
:meth:`loop.set_default_executor`
```
both render the same exact way.

Reply:
> Edit: Wait nevermind, I see you were referring to the extra "asyncio" part. Personally, I think it might be needed here since there's no mention that the entry belongs to the asyncio module otherwise. I do agree that it normally would look better to have loop.set_default_executor though.

Yes, because there's no such thing as `asyncio.loop.something()`.  People will write `loop.something()` in their code.

Reply:
Okay, I'll change it as suggested then.

Reply:
Thank you! Sorry for nitpicking, just trying to make sure we ship the best thing! :)

Reply:
> Thank you! Sorry for nitpicking, just trying to make sure we ship the best thing! :)

No problem. I'd much rather that we nitpick and discuss every change than just merge PRs without question, it's worth the improvement in quality. (:

Reply:
@1st1 Done, I think that's everything.